MekHQ/src/mekhq/gui/enums/PersonnelTableModelColumn.java

```diff
@@ -877,10 +877,14 @@ public boolean isVisible(final Campaign campaign, final PersonnelTabView view,
             case ORIGIN_PLANET:
             case PORTRAIT_PATH:
                 return new NaturalOrderComparator();
-            case SALARY:
-                return new FormattedNumberSorter();
-            case SKILL_LEVEL:
```

Comment:
What happened to SKILL_LEVEL?

Reply:
Nothing happened, no accidental removal at all...